app/src/main/java/org/gdg/frisbee/android/common/GdgActivity.java

```diff
@@ -130,10 +130,7 @@ protected void onStart() {
 
         mGoogleApiClient.registerConnectionCallbacks(this);
         mGoogleApiClient.registerConnectionFailedListener(this);
-
-        if (PrefUtils.isSignedIn(this)) {
-            mGoogleApiClient.connect();
-        }
+        mGoogleApiClient.connect();
```

Comment:
This introduced problems too. I've cleared data and setup the app without Signin. 
Since Google knows me and I have this app signed in elsewhere, because of this connect call, it automatically signed in. 
You may say that it is good but our app is not designed for that. The drawback is that `PrefUtils.isSignedIn()` stayed false eventhough I have signed in. The app thinks that I am not signed in.